src/templates/static/platforms.html.tt

```diff
@@ -53,138 +53,138 @@
-                                <p>[% l("[_1] enters Forex and CFD trading with the universally acclaimed MetaTrader 5 to its continued leadership presence in the binary options market.", website_name) %]</p>
-                                <p>[% l("Trade Forex and Contracts for Difference (CFD) assets with highly-competitive leverage and zero commission.") %]</p>
-                                <p>
-                                    <a class="button" href="[% request.url_for('user/metatrader') %]">
-                                        <span>[% l("Create MetaTrader 5 Account in [_1]", website_name) %]</span>
-                                    </a>
-                                </p>
-                            </div>
+                            [% INCLUDE platform image_class='gr-12 gr-7-p gr-10-m' image='trading-page' header=website_name description=l("Premier binary options trading platform")
+                                text=l("Trade in the world’s financial markets with a simple and user-friendly online platform.")
+                                url=request.url_for('trading') button_text=l("Trade Now") %]
+                            <div class="gr-2"></div>
+                            [% INCLUDE platform image_class='gr-7 gr-5-p gr-7-m' image='tick-trade' header=l("Binary Tick Trade App") description=l("Ultra fast on-the-go trading")
+                                text=l("Enjoy our fastest type of trading with our Tick Trade app, wherever you are.")
+                                app_store_url='https://itunes.apple.com/app/apple-store/id1091187567?pt=117911266&ct=platforms_page&mt=8'
+                                google_play_url='https://play.google.com/store/apps/details?id=com.binary.ticktrade&referrer=utm_source%3Dbinary-com%26utm_medium%3Dreferrer%26utm_campaign%3Dplatforms_page' %]
+                        </div>
+                        <div class="gr-row gr-padding-30">
+                            [% INCLUDE platform image_class='gr-8 gr-5-p gr-7-m' image='webtrader' header=l("Binary Webtrader") description=l("Advanced binary options trading interface")
+                                text=l("Monitor the movements of your favourite assets and markets at the same time.")
+                                url='https://webtrader.binary.com' target='_blank' button_text=l("Try Webtrader") %]
+                            <div class="gr-2"></div>
+                            [% INCLUDE platform image_class='gr-12 gr-7-p gr-10-m' image='nextgen' header=l("Binary Next-Gen") description=l("Advanced trading app for web and mobile")
+                                text=l("Explore advanced trading features with the Next-Gen app for web and mobile.")
+                                url='https://app.binary.com' target='_blank' button_text=l("Try Next-Gen")
+                                app_store_url='https://itunes.apple.com/app/apple-store/id1134884301?pt=117911266&ct=platforms_page&mt=8'
```

Comment:
should we remove the link to next-gen apple store? if i remember correctly, the next-gen app was removed from the apple store

Reply:
yes. please remove iOS links to next-gen and tick trade